### PR TITLE
Ensure that always enough events are cached

### PR DESCRIPTION
**Pull request by leo**
We only need to ensure that about 10 events are renderable for each group (Me, Team, System), because the view only has space for about half of that.

### CODE REVIEW

File: renderer/pages/feed.js [hunk 1]
```diff
@@ -508,7 +508,9 @@ class Feed extends React.Component {
     for (const group of groups) {
       const { length } = this.filterEvents(scopedEvents, relatedTeam, group)
 
-      if (length > 30) {
+      // Ensure that always at least 10 events
+      // are cached for each event group
+      if (length > 10) {
```
Comment (nkzawa) [inline renderer/pages/feed.js]: it's not important at all but it should be `>= 10` if it's `at least 10 events` ?

**Comment by matheuss** [comment]
sgtm but not sure how it’ll impact infinte scrolling

**Comment by leo** [comment]
@matheuss Not at all

File: renderer/pages/feed.js (hunk 1 above)
Comment (leo) [inline renderer/pages/feed.js]: 👍

**Pull request merged**